backend/src/activitypub/ActivityPub.js

```diff
@@ -22,16 +22,16 @@ let activityPub = null
 export { activityPub }
 
 export default class ActivityPub {
-  constructor (activityPubEndpointUri, internalGraphQlUri) {
-    this.endpoint = activityPubEndpointUri
+  constructor (internalGraphQlUri) {
+    this.endpoint = internalGraphQlUri
```

Comment:
@Mastercuber are you sure? `GRAPHQL_URI` is an cluster-internal endpoint not exposed to the internet.

![Screenshot (69)](https://user-images.githubusercontent.com/2110676/55442986-30559c00-55b1-11e9-9e95-779a72311dc7.png)

Reply:
@roschaefer oh I remember, so there is a switch needed for using the internal endpoint for development and the other endpoint for production. Maybe when starting the Server in dev mode, an environment var could also be set?

Reply:
Is it required? We proxy `/api` to `GRAPHQL_URI`. So if the backend knows `CLIENT_URI` the publicly available graphql endpoint is `$CLIENT_URI/api`. Does that help?

I still don't understand this PR by the way.

Reply:
Initially it was a fix for serving paged collections. Yes it helped, I deleted the second commit. So this is again a quick fix. 

By the way a recognized that on the staging not only the `/api/activitypub` endpoint, but also the `/activitypub` endpoint is exposed.

Reply:
*and* `/.well-known`